ArmPkg/Library/ArmMmuLib/AArch64/ArmMmuLibPageTableAlloc.c

```diff
@@ -22,20 +22,20 @@ PAGE_TABLE_MEM_ALLOC_PROTOCOL  *mPageTableMemAllocProtocol = NULL;
   @return A pointer to the allocated buffer or NULL if allocation fails
 **/
 VOID *
-AllocatePageTableMemory (
+MmuAllocatePageTableMemory (

```

Comment:
@os-d @kuqin12 Is this function supposed to be consumed in ArmMmuLibPageTableAlloc.c? Because that file has a the function declaration for `AllocatePageTableMemory` but it's not clear what its source is supposed to be. 
If this function's name is `AllocatePageTableMemory` there is a linking error: 

> error LNK2005: AllocatePageTableMemory already defined in ArmCpuDxe.lib(PageTableMemoryAllocation.obj)
INFO - C:\devbox\hyperv.uefi\Build\MsvmAARCH64\RELEASE_VS2022\AARCH64\ArmPkg\Drivers\CpuDxe\CpuDxe\DEBUG\ArmCpuDxe.dll.wtemp : fatal error LNK1169: one or more multiply defined symbols found

I previously made the function static but that prevents it from being consumed since it's not published as a protocol.

Reply:
This seems like a naming conflict, did the log say which other function you are clashing with?

Reply:
You should be able to just make the one in CpuDxe `STATIC`, right? It looks like that is what the conflict is. The CpuDxe one should not be directly called outside of the file, only through the protocol

Reply:
Making the function static in CpuDxe worked to fix that conflict.